src/gui/plugins/resource_spawner/ResourceSpawner.cc

```diff
@@ -739,7 +739,7 @@ void ResourceSpawner::RunFetchResourceListThread(const std::string &_owner)
             resource.isFuel = true;
             resource.isDownloaded = false;
             resource.owner = id.Owner();
-            resource.sdfPath = id.UniqueName();
+            resource.sdfPath = "https://" + id.UniqueName();
```

Comment:
Can we avoid hard coding `https://`? Maybe get it from `server` somehow?

Reply:
Actually looking at this again, the `UniqueName` won't work on Windows (though gui plugins currently don't have any test coverage).  Let me re-work it.

Reply:
Okay, I ended up adding a `Url` method to the Identifier objects in fuel_tools.  This now depends on https://github.com/gazebosim/gz-fuel-tools/pull/429